scss/_detail.scss

```diff
@@ -14,8 +14,17 @@
   display: flex;
   max-width: 650px;
   padding: 16px;
+  position: relative;
   width: 100%;
 
+  &::before {
+    bottom: 50px;
+    content: url('./assets/badge.png');
```

Comment:
You should use` background-image `attribute and `content: ""`

Reply:
Done 😄